src/service/user-service.ts

```diff
@@ -184,6 +189,35 @@ export default class UserService {
     return Promise.resolve(this.getSingleUser(user.id));
   }
 
+  /**
+   * Closes a user account by setting the user's status to deleted and inactive.
+   * Also, it sets the user's acceptedToS status to NOT_ACCEPTED and canGoIntoDebt to false.
+   * After saving these changes, it sends an account closure notification email to the user.
+   *
+   * @param {number} userId - The ID of the user to close the account for.
+   * @returns {Promise<void>} - A promise that resolves when the user account has been closed and the email has been sent.
+   * @throws {Error} - Throws an error if the email could not be sent.
+   */
+  public static async closeUser(userId: number): Promise<UserResponse> {
+    const user = await User.findOne({ where: { id: userId, deleted: false } });
+    if (!user) return undefined;
+    const currentBalance = await BalanceService.getBalance(user.id);
+
+    user.deleted = true;
+    user.active = false;
+    user.acceptedToS = TermsOfServiceStatus.NOT_ACCEPTED;
```

Comment:
Ik vind niet dat we dit moeten veranderen, want als hun account weer wordt geheractiveerd dan moeten ze wéér de ToS accepteren.

Reply:
sure